blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -187,7 +187,7 @@ class RunDetails extends Component {
                             <TabLink to="/artifacts">{t('rundetail.header.tab.artifacts', {
                                 defaultValue: 'Artifacts',
                             })}</TabLink>
-                        </PageTabs>
+                        </PageTabsOld>
```

Comment:
I did a full search/replace for now, so no code except the *new* header components should be using the new `PageTabs`, and once the components have been merged into actual pages in place of the old headers we'll get rid of `PageTabsOld` from JDL completely.